packages/webpack-plugin/package.json

```
@@ -19,9 +19,29 @@
   "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
   "exports": {
-    "types": "./dist/index.d.ts",
-    "import": "./dist/index.mjs",
-    "require": "./dist/index.js"
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./loader": {
+      "types": "./dist/loader.d.ts",
+      "require": "./dist/loader.js",
+      "import": "./dist/loader.mjs"
+    },
+    "./cssLoader": {
+      "types": "./dist/cssLoader.d.ts",
+      "require": "./dist/cssLoader.js",
+      "import": "./dist/cssLoader.mjs"
+    },
+    "./assets/kuma.css": "./assets/kuma.css"
```

The additional conditional exports are intended for internal implementation purposes only, so personally, I prefer not to disclose them casually. The reason is that making changes to these definitions in the future could result in disruptive modifications. I believe it would be better to either remove them or change the path in a way that conveys their internal use to the users.

```suggestion
    "./_internal/loader": {
      "types": "./dist/loader.d.ts",
      "require": "./dist/loader.js",
      "import": "./dist/loader.mjs"
    },
    "./_internal/cssLoader": {
      "types": "./dist/cssLoader.d.ts",
      "require": "./dist/cssLoader.js",
      "import": "./dist/cssLoader.mjs"
    },
    "./_internal/assets/kuma.css": "./assets/kuma.css"
```